tests/flag_util_test.py

```diff
@@ -132,5 +133,18 @@ def testReadAndWrite(self):
     self.assertFlagState(flag_values_copy, 2, True)
 
 
+class TestUnitsParser(unittest.TestCase):
+  def setUp(self):
+    self.up = flag_util.UnitsParser()
+
+  def testParser(self):
+    self.assertEqual(self.up.Parse('10KiBo'),
+                     10 * 1024 * pint_wrapper.ureg.bytes)
```

Comment:
Are KiBo and bytes built into pint? I saw a comment earlier giving "12MB" as an example, which seems to conflict with this?